server/rpc_server.go

```diff
@@ -41,6 +55,64 @@ func NewRPCServer(security config.Security) *grpc.Server {
 	if s == nil {
 		s = grpc.NewServer()
 	}
-	diagnosticspb.RegisterDiagnosticsServer(s, &sysutil.DiagnoseServer{})
+	globalDomain = dom
+	// For redirection the cop task.
+	mocktikv.TiDBRPCServerCoprocessorHandler = HandleCopDAGRequest
```

Comment:
Is it necessary to redirect?

Reply:
When only `tidb-server` with mockTiKV, we need to redirect the request. The client of mockTiKV not use `grpc client` to send request.

Reply:
I think it's better to add a field to mockTiKV instead of defining a global variable.